tests/ml/ml_test.py

```diff
@@ -536,7 +536,12 @@ def test_kbinsdiscretizer(tmpdir, strategy):
                                y=[1, 2, 5, 6, 10, 9])
 
     trans = vaex.ml.KBinsDiscretizer(features=['x', 'y'], n_bins=3, strategy=strategy)
-    df_train_trans = trans.fit_transform(df_train)
+
+    if strategy == 'quantile':
+        with pytest.warns(UserWarning):
```

Comment:
Please use match to make sure it's giving an expected warning.

Reply:
good idea - done!